packages/superset-ui-chart/src/types/Base.ts

```diff
@@ -0,0 +1,5 @@
+export type HandlerFunction = (...args: any[]) => void;
```

Comment:
these could also just live in the `index` if they are generic/general? regardless, I don't think they're exported from the `index` now/not sure if you want them to be?

Reply:
They are internal to this package now. I am trying not to export too many types from index if not necessary.